src/ScenarioState/Exception/MissingStateException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/*
+ * This file is part of the ScenarioStateBehatExtension project.
+ *
+ * (c) Rodrigue Villetard <rodrigue.villetard@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Gorghoa\ScenarioStateBehatExtension\ScenarioState\Exception;
+
+/**
+ * Class MissingStateException
+ * @package Gorghoa\ScenarioStateBehatExtension\ScenarioState\Exception
```

Comment:
Useless `@package` annotation.
Can you also add a `@author` annotation with your name please?